acceptance/current/.acceptance/acceptance-cookbook/recipes/destroy.rb

```diff
@@ -0,0 +1,3 @@
+execute 'kitchen destroy' do
+  cwd File.join(File.dirname(__FILE__), "../../../../../../../..")
```

Comment:
+1 for what @patrick-wright is saying. Once we have a few projects using chef-acceptance and we know what they need, we will introduce shortcuts.